tests/pages/Tool.test.tsx

```diff
@@ -1,6 +1,6 @@
 // you want to import from test-utils instead of testing-library/react since we overwrote the render function to support our wrapper providers
 import { render, screen } from '../test-utils';
-import Tool, { getStaticPaths, getStaticProps } from '../../pages/tool/[id]';
+import Tool from '../../pages/tool/[id]';
```

Comment:
Were you not going to test the `getServersideProps`?

Reply:
You're right I should test that. I'll update it